src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/domain/VedtaksperiodeUtil.kt

```diff
@@ -8,14 +8,34 @@ import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeil
 import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
 import no.nav.tilleggsstonader.sak.util.formatertPeriodeNorskFormat
 import no.nav.tilleggsstonader.sak.vedtak.domain.StønadsperiodeBeregningsgrunnlag
+import java.time.LocalDate
 
 object VedtaksperiodeUtil {
     fun validerVedtaksperioder(
         vedtaksperioder: List<Vedtaksperiode>,
+        vedtaksperioderForrigeBehandling: List<Vedtaksperiode>?,
         stønadsperioder: List<StønadsperiodeBeregningsgrunnlag>,
+        revurderFra: LocalDate?,
     ) {
         validerIngenOverlappendeVedtaksperioder(vedtaksperioder)
         validerVedtaksperiodeOmfattesAvStønadsperioder(vedtaksperioder, stønadsperioder)
+        validerIngenEndringerFørRevurderFra(vedtaksperioder, vedtaksperioderForrigeBehandling, revurderFra)
+    }
+
+    fun validerIngenEndringerFørRevurderFra(
+        vedtaksperioder: List<Vedtaksperiode>,
+        vedtaksperioderForrigeBehandling: List<Vedtaksperiode>?,
+        revurderFra: LocalDate?,
+    ) {
+        if (revurderFra == null || vedtaksperioderForrigeBehandling.isNullOrEmpty()) return
```

Comment:
Hvis man har:
Behandling 1: Vedtaksperiode 1-31 jan
Behandling 2: Opphør 1 jan (alle vedtaksperioder slettes)
Behandling 3: Revurderer fra og med 1 feb, der man legger inn perioder for januar

Så vil `orEmpty` her ikke validere at man har lagt inn perioder før revurder fra

Reply:
Det gjorde det en del mer komplekst, men tror jeg fikk det til! 0fb9c22c9b08e5fc12d7ac701a75807c4711cf13